Source/DafnyLanguageServer/Workspace/GutterIconAndHoverVerificationDetailsManager.cs

```diff
@@ -309,13 +309,40 @@ public class GutterIconAndHoverVerificationDetailsManager : IGutterIconAndHoverV
           targetMethodNode.StatusVerification = finalOutcome;
         }
 
+        var options = compilation.Program.Reporter.Options;
+        ReportVacuityAndRedundantAssumptionsChecks(compilation, implementation, verificationResult, options);
```

Comment:
Could you move this out of this class, to where `ReportEndVerifyImplementation` is called?